GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1233,7 +1233,7 @@ private void SelectInitialRevision()
                 }
                 else
                 {
-                    toBeSelectedObjectIds = new ObjectId[] { Module.GetCurrentCheckout() };
+                    toBeSelectedObjectIds = currentCheckout is null ? Array.Empty<ObjectId>() : new ObjectId[] { currentCheckout };
```

Comment:
This change is from #8846 (where I first applied the changes) for null annotation but makes sense here.